ansible_collections/arista/avd/roles/eos_cli_config_gen/README_v4.0.md

```diff
@@ -2487,7 +2487,7 @@ ip_routing_ipv6_interfaces: < true | false >
 ```yaml
 router_general:
   vrfs:
-    < destination-vrf >:
+    - destination: < destination-vrf >
```

Comment:
Please rename this key to ```destination_vrf``` because we are using ```source_vrf``` as a key below (to define source vrfs).

Reply:
fiixed